Tasks/AppCenterTest/package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "vsts-tasks-vsmobilecentertest",
+  "name": "vsts-tasks-vsappcentertest",
```

Comment:
It appears that we've taken the 'vs' off the front of all the other references to 'app center'.  Remove it here to be consistent?